web/api/js/codechecker-api-node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "codechecker-api",
-  "version": "6.39.0",
+  "version": "6.40.1",
```

Comment:
I needed to upgrade the version to `40.1` for getting the change reflected. Hope that is alright!

Reply:
Why not just upgrade to 6.40.0? Is there a reason to update the minor version and the patchlevel version at once? Probably only a minor version from 6.39.0 to 6.40.0 is really required, right?

Reply:
@bulwahn we did that, but due to some error in the `.thrift` file had to change it. It wasn't quite working with the version `6.40.0`

Reply:
Well, fortunately( or unfortunately) the version in the original repository is now updated to `6.40.0` so ours should be an update to it. 

Maybe @csordasmarton can help tell what we need to go in this case. Should we keep `6.40.1` or revert back to `6.40.0`

Reply:
@jay24rajput Sorry, I'm a little bit busy in these days. The current api version is `6.40.0`, so you have to use `6.41.0`. Try to regenerate the zip files in the following way: https://github.com/Ericsson/codechecker/tree/master/docs/web/api.

If a package is already installed on your local system, maybe pip/npm is caching those, so this can be a reason why you don't see your updates when you generate zip files multiple files with the same version. As soon as I have more free time I will check it.

In the zip files you don't have to resolve the merge conflicts, just rebase your branch, resolve the merge conflicts and regenerate the zip files again.